frontend/src/component/common/PremiumFeature/PremiumFeature.tsx

```diff
@@ -108,6 +108,11 @@ const PremiumFeatures = {
         url: 'https://docs.getunleash.io/reference/banners',
         label: 'Banners',
     },
+    'incoming-webhooks': {
+        plan: FeaturePlan.ENTERPRISE,
+        url: 'https://docs.getunleash.io/reference/incoming-webhooks',
```

Comment:
We need to double check these URLs once we tackle the docs. Added this reference to https://linear.app/unleash/issue/2-1702/beta-document-the-new-incoming-webhooks-feature so we don't forget.